app/mobile/BuPazar/app/src/main/java/com/example/bupazar/page/activity/login/LoginActivity.kt

```diff
@@ -54,11 +61,45 @@ class LoginActivity : BaseActivity(), ActivityChangeListener {
             startActivity(intent)
         }
 
+        textViewForgotPassword.setOnClickListener(){
+            setContentView(R.layout.fragment_forgot_password)
+            resetButton.setOnClickListener(){
+
+                if (mailView.text.isEmpty()){
+
+                    Toast.makeText(this@LoginActivity,"Please type your email address!", Toast.LENGTH_SHORT).show()
+                }
+                else {
+
+                    val apiService = RestApiService()
+                    val forgotPassword = ForgotPasswordRequest(
+                        userMail = mailView.text.toString(),
+                    )
+                    apiService.forgotPassword(forgotPassword) {
+                        // TODO: response back as null email cannot be checked.
```

Comment:
I created new issue #284  to solve this. After backend fix this, we can add as a new feature. Other areas are correct